src/lib/MongoQueue.ts

```diff
@@ -0,0 +1,94 @@
+import { Model, Schema } from 'mongoose';
+import { Queue, Media, QueueManager } from 'musicca';
+
+export interface IQueue {
+  _id: string;
+  list: Media[];
+}
+
+export const schema = new Schema<IQueue>({
+  _id: String,
+  list: {
+    type: [Object],
+    default: []
+  }
+});
+
+export default class MongoQueue extends Queue {
+  public db: Model<IQueue>;
+
+  public id: string;
+
+  constructor(manager: QueueManager<Queue>, id: string, { model }: { model: Model<IQueue> }) {
```

Comment:
I would recommend moving the `options` type to be its own interface rather than setting it inline.